README.md

```diff
@@ -46,6 +48,29 @@ module.exports = {
 }
 ```
 
+If you need to use options with `css-loader`.
+
+```js
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ExtractTextPlugin.extract({
+          fallback: "style-loader",
+          use: 'css-loader?modules,localIdentName="[name]-[local]-[hash:base64:6]"'
```

Comment:
I guess the question is why it doesn't work for @pragyandas. I would need a project to study to say why.